index.html

```diff
@@ -326,7 +326,7 @@ <h4 class="functionality__paragraph">Necesito Ayuda</h4>
                 <a href="#" class="functionality__Saber-mas">Saber más</a>
               </div>
             </div>
-          </div>
+          </div>  
```

Comment:
Creo que esto no hace algo
```suggestion
          </div>
```